embedded-graphics/src/lib.rs

```diff
@@ -174,20 +176,33 @@ pub mod primitives;
 pub mod style;
 pub mod transform;
 
-use crate::geometry::Dimensions;
+use crate::geometry::{Dimensions, Point, Size};
 use crate::pixelcolor::PixelColor;
+use crate::prelude::*;
 
-/// To use this crate in a driver, `Drawing` must be implemented. This allows display drivers to
-/// support all embedded_graphics objects through the `draw()` method.
+/// Defines a display that can be used to render [`Drawable'] objects.
+///
+/// To use this crate in a driver, `DrawTarget` must be implemented. This trait defines how a
+/// display draws pixels, and optionally provides a way to define accelerated drawing methods for
+/// graphical primitives such as lines, rectangles, triangles, and circles.
+///
+/// Once a `DrawTarget` is defined, it can be used to render [`Drawable`]. Note that any iterator
```

Comment:
Typo: You removed the last word of the sentence when you changed the link.
```suggestion
/// Once a `DrawTarget` is defined, it can be used to render [`Drawable`] objects. Note that any iterator
```
or maybe
```suggestion
/// Once a `DrawTarget` is defined, it can be used to render [`Drawable`]s. Note that any iterator
```